src/app/lib/config/services/serbian.js

```diff
@@ -38,7 +35,6 @@ const service = {
     home: 'Home',
     currentPage: 'Current page',
     skipLinkText: 'Skip to content',
-    relatedContent: 'Related content',
```

Comment:
Since you've pulled in `latest` some fields have changed in the serbian config. Can you please change them to match the `latest` branch? 
`relatedContent` and `mostRead` fields are missing and two footer links were added back in again which should be removed.